File: poc-challenges/250803/src/test/java/com/tdd/domain/FactorialTest.java
package com.tdd.domain;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class FactorialTest {
	@Test
	void factorial_test() {
		assertEquals(1, fac(1));
	}

	private int fac(int n) {
		return 0;
	}
}

File: poc-challenges/260104/src/test/kotlin/com/currenjin/tdd/FactorialTest.kt
package com.currenjin.tdd

import org.junit.jupiter.params.ParameterizedTest
import org.junit.jupiter.params.provider.CsvSource
import kotlin.test.assertEquals

class FactorialTest {
    @ParameterizedTest
    @CsvSource(value = ["1,1", "2,2", "3,6"])
    fun factorial_test(input: Int, expected: Int) {
        assertEquals(expected, Factorial.fac(input))
    }
}